src/test/java/com/checkout/disputes/DisputesTestIT.java

```diff
@@ -98,8 +99,8 @@ void shouldFailOnAcceptDisputeAlreadyAccepted() {
      * This test is disabled to avoid long waiting due the async operations that requires to perform a dispute
      * however it complete functional and can be enabled for confirmation purposes
      */
-    //@Test
-    //@Timeout(30)
+    @Test
+    @Timeout(30)
```

Comment:
The test failed because we need like about 5 minutes to run. I let this test as disabled because every time we send a file to the server, as I was not sure about if there are any policy for the file storage in Sandbox I decided to disable the test.